crates/holochain/tests/inline_zome_sketch.rs

```diff
@@ -132,3 +133,33 @@ async fn inline_zome_3_agents_2_dnas() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test(threaded_scheduler)]
+#[cfg(feature = "test_utils")]
+async fn invalid_cell() -> anyhow::Result<()> {
```

Comment:
Even though it passes, should this test be ignored to denote that it needs to be completed?

```suggestion
#[tokio::test(threaded_scheduler)]
#[cfg(feature = "test_utils")]
#[ignore("Needs to be completed when HolochainP2pEvents is accessible")]
async fn invalid_cell() -> anyhow::Result<()> {
```